src/libraries/tests.proj

```diff
@@ -112,6 +112,11 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Drawing.Common\tests\System.Drawing.Common.Tests.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetOS)' != 'Windows'">
```

Comment:
I think we should do this in the test code instead:

https://github.com/dotnet/runtime/blob/6ffb7f0504c8be8e54ba5ad55a04c4635ec49275/src/libraries/Microsoft.Extensions.Hosting.WindowsServices/tests/UseWindowsServiceTests.cs#L12-L13

That way someone can `dotnet test` on a non-Windows machine and it "passes".

Reply:
I ask because we recently had https://github.com/dotnet/runtime/issues/65210

Reply:
I will add the `ConditionalFact` *also*, so local `dotnet test` on non-windows machines won't fail.